crates/proof-of-sql/src/sql/proof_plans/projection_exec.rs

```diff
@@ -71,15 +87,15 @@ impl ProofPlan for ProjectionExec {
     }
 
     fn get_column_references(&self) -> IndexSet<ColumnRef> {
-        let mut columns = IndexSet::default();
+        let mut columns = self.input.get_column_references();
```

Comment:
```suggestion
        return self.input.get_column_references();
```

Reply:
You are right. Projection should result in all output column refs also being input ones.